bin/node/src/service.rs

```diff
@@ -151,10 +151,10 @@ pub fn new_full(mut config: Configuration) -> Result<TaskManager, ServiceError>
         other: (block_import, justification_rx, mut telemetry, metrics),
     } = new_partial(&config)?;
 
-    config
-        .network
-        .extra_sets
-        .push(finality_aleph::peers_set_config());
+    let mut peers_set_config = finality_aleph::peers_set_config();
+    peers_set_config.set_config.reserved_nodes =
+        config.network.default_peers_set.reserved_nodes.clone();
+    config.network.extra_sets.push(peers_set_config);
```

Comment:
That `mut` here is kinda awkward. Can we pass a `Vec<MultiaddrWithPeerId>` as a parameter to `peers_set_config`?

Reply:
Done.

Reply:
`finality_aleph::peers_set_config();` seems to be doing `SetConfig::default()` modulo the number of in/out of peers:
https://github.com/paritytech/substrate/blob/nil/client/network/src/config.rs#L556

Reply:
We need to set our name, despite that we do nothing as apparently it hasn't had any effect :(

Reply:
we could still comply with the note though, something along the lines of:
```
pub fn peers_set_config() -> sc_network::config::NonDefaultSetConfig {
    let mut non_default_set = sc_network::config::NonDefaultSetConfig::new(
        network::ALEPH_PROTOCOL_NAME.into(),
        // max_notification_size should be larger than the maximum possible honest message size (in bytes).
        // Max size of alert is UNIT_SIZE * MAX_UNITS_IN_ALERT ~ 100 * 5000 = 50000 bytes
        // Max size of parents response UNIT_SIZE * N_MEMBERS ~ 100 * N_MEMBERS
        // When adding other (large) message types we need to make sure this limit is fine.
        1024 * 1024,
    );

    non_default_set.set_config = sc_network::config::SetConfig::default();
    non_default_set.set_config
}
```

Reply:
done.